Libplanet.Tests/Blockchain/BlockChainTest.cs

```diff
@@ -279,6 +280,21 @@ public void EvaluateActions()
             Assert.Equal(states[TestEvaluateAction.BlockIndexKey], blockIndex);
         }
 
+        [Fact]
+        public void CanDetectInvalidActionType()
```

Comment:
Could you omit the prefix `Can`?  We've recently started to omit that prefix.  See also [this](https://github.com/planetarium/libplanet/pull/117#pullrequestreview-213330287) and [this](https://github.com/planetarium/libplanet/pull/117#issuecomment-472338131).